pkgs/development/libraries/gtk/3.x.nix

```
@@ -48,13 +48,11 @@
 
 assert cupsSupport -> cups != null;
```

```suggestion
```

also the cups ? null can be removed.

---

I think this is to fail fast on systems that `null` `cups` in overlays or something.